codespell_lib/data/dictionary.txt

```diff
@@ -20490,6 +20506,7 @@ hearbeating->heartbeating, heart beating, hear beating,
 hearbeats->heartbeats, heart beats, hear beats,
 heared->heard, header,
 heathy->healthy
+heder->header
```

Comment:
Could also be `herder`.

Reply:
This typo is very rare. I will add it anyways.